Duckling/Numeral/PL/Rules.hs

```diff
@@ -60,6 +60,33 @@ ruleSixty = Rule
   , prod = \_ -> integer 60
   }
 
+ruleSeventy :: Rule
+ruleSeventy = Rule
+  { name = "seventy"
+  , pattern =
+    [ regex "siedemdziesi(ą)t|siedemdziesi(ę)ci(u|oma)"
```

Comment:
I would add support for the case without polish letters, for example, if the word would be "siedemdziesiat", also you don't have to write "or" in regex and paste the repeating part of the word ("siedemdziesi") again, you could group the part of the word that could be viewed differently, instead of the whole word.
I would suggest to replace regex with: "siedemdziesi((a|ą)t|ęci(u|oma))"